FireRender.Maya.Src/FireRenderSurfaceOverride.cpp

```diff
@@ -242,8 +253,11 @@ bool FireRenderStandardMaterialNodeOverride::valueChangeRequiresFragmentRebuild(
 	{
 		// Check if any attribute which affects results of getCustomMappings() is changed.
 		MString pname = plug->partialName(false, false, false, false, true, true);
-		if (pname == "diffuse" || pname == "emissive" || pname == "reflections")
+		if (pname == "diffuse" || pname == "emissive" || pname == "reflections" ||
+			pname == "transparencyEnable" || pname == "useTransparency3Components")
```

Comment:
Shouldn't it be && instead of || ?
Maybe I got it wrong but shouldn't "transparency enabled" go together with "useTransparency3Components"

Reply:
How "pname" can be equal to different values at the same time ?